src/hooks/useSelectedNetwork.ts

```diff
@@ -20,6 +21,9 @@ export const useSelectedNetwork = (): {
   const { disconnect } = useAccounts();
   const selectedNetwork = useSelector(getSelectedNetwork);
 
+  const { wallets } = useWallets();
+  const wallet = wallets[0];
```

Comment:
Can you replace L25 with:
```
const wallet = wallets.find((wallet) => wallet.walletClientType === 'privy');
```

This will just guarantee it's the embedded wallet (that should never not be the case but just want to make this future proof)